installer/scripts/genRubyInstaller.sh

```diff
@@ -28,16 +28,16 @@ substitute_arch()
     echo -n "$RESULT"
 }
 
-# This script will generate a "starting point" for an InstallBuilder data file.
-# For example, this generated ruby.data in the omsagent project. This is useful
-# for a package that has an install process (i.e. make install), but we need to
-# generate an install data file for it.
+# This script will generate a file that will contain the new
+# ruby  changes for ruby.data file based on SOURCE_DIR content to be included in ruby install builder 
+# since building ruby has an install process
+# and requires generating an install ruby data file for it.
```

Comment:
That's a heck of a run-on sentence, and is hard to understand.

How about replacing lines 31-34 with something like:

# This script will generate a Ruby.data file, the file that is consumed
# by InstallBuilder, to build a native RPM or DEB installation package.
#
# It is necessary to regenerate this file if new packages have been
# installed into Ruby, and those new packages should be deployed
# on customer systems during OMS installation.

I think this makes more sense. What do you think?